board/boards/black.h

```diff
@@ -206,7 +209,7 @@ const board board_black = {
   .has_hw_gmlan = false,
   .has_obd = true,
   .has_lin = false,
-  .has_rtc = false,
+  .has_rtc = true,
```

Comment:
would it make more sense to keep the old values and rename it to `has_rtc_battery`? Technically they all have RTC, so storing that in the struct isn't useful of course

Reply:
that way you can also check this field in the `rtc_init()` check adeeb was complaining about

Reply:
Good point! +